query-planner-js/src/buildPlan.ts

```diff
@@ -105,6 +105,7 @@ import { Conditions, conditionsOfSelectionSet, isConstantCondition, mergeConditi
 import { enforceQueryPlannerConfigDefaults, QueryPlannerConfig, validateQueryPlannerConfig } from "./config";
 import { generateAllPlansAndFindBest } from "./generateAllPlans";
 import { QueryPlan, ResponsePath, SequenceNode, PlanNode, ParallelNode, FetchNode, SubscriptionNode, trimSelectionNodes } from "./QueryPlan";
+import { performance } from 'perf_hooks';
```

Comment:
guess this is a leftover from previous changes?

Reply:
No, this is to be able to use `performance.now()`